### DISCUSSION TITLE
In readme file i seen CreateBond() and isBonded()  are not supported for IOS device . So what else i can use instead of that to replace that feature.

### DISCUSSION THREAD

**Discussion by madhan-SEET** [capacitor-community/bluetooth-le#671] [Q&A] [answered]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Plugin version:**
- @capacitor-community/bluetooth-le: [e.g. 0.4.0]

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by pwespi** [comment, marked as answer]
On iOS bonding is handled by the OS. When accessing a characteristic that requires bonding the OS will trigger the bonding process. Or it can be done by the user in the OS Bluetooth settings.
